app/models/user.rb

```diff
@@ -1,8 +1,30 @@
 class User<ApplicationRecord
```

Comment:
Intendation Fix: `class User < ApplicationRecord`
Also check for intendation at other places as well.